### DISCUSSION TITLE
Questions on using the power down mode

### DISCUSSION THREAD

**Discussion by nabelekt** [SpenceKonde/DxCore#163] [Q&A] [answered]
Somewhat related to #161, but not necessarily an issue:

1. From the example in [PowerSave.md](https://github.com/SpenceKonde/DxCore/blob/master/megaavr/extras/PowerSave.md):
```
RTC.PITCTRLA = RTC_PERIOD_CYC16384_gc /* RTC Clock Cycles 16384, resulting in 32.768kHz/16384 = 2Hz */
```
implies to me that the MCU will sleep for ~500ms (wake at 2 Hz).
- Is that the case?
- How can the sleep time be customized?

2. One of the suggested ways of eliminating issues caused by floating pins is:
```
The input buffer is disabled in the PORTx.PINnCTRL register (a wrapper function around this is planned for a future update)
```
Has that wrapper function been written yet, or is that still planned?

Thanks!

**Comment by nabelekt** [comment]
@SpenceKonde answered these questions here: https://github.com/SpenceKonde/DxCore/issues/161#issuecomment-948113396

**Comment by SpenceKonde** [comment, marked as answer]
I wouldn't say I answered much there... 

1. Yeah it looks like it - There's a `RTC_PERIOD_CYC32768_gc` option, and it looks to me like the RTC prescaler that can be set in RTC.CTRLA does effect the PIT clock... 

2. pinConfigure() provides a wrapper around disabling pins, the latest documentation also gives a quick description of the multipin configuration registers (no wrapper - it doesn't really lend itself to that) which can be used to disable inputs on large numbers of pins quickly and easily.

**Comment by nabelekt** [comment]
@SpenceKonde, will https://github.com/SpenceKonde/megaTinyCore/issues/158 apply to DxCore as well?

**Reply by SpenceKonde** [reply]
yes
